src/env.js

```diff
@@ -205,6 +208,7 @@ Sk.configure = function (options) {
     Sk.switch_version("round$", Sk.__future__.dunder_round);
     Sk.switch_version("next$", Sk.__future__.dunder_next);
     Sk.switch_version("clear$", Sk.__future__.list_clear);
```

Comment:
Change this to use `python3`
```suggestion
    Sk.switch_version("clear$", Sk.__future__.python3);
```